packages/odyssey-storybook/src/components/odyssey-mui/CssBaseline/CssBaseline.stories.tsx

```diff
@@ -16,7 +16,7 @@ import {
   CssBaselineProps,
   ScopedCssBaseline,
   ScopedCssBaselineProps,
-} from "@okta/odyssey-react-mui";
```

Comment:
We should keep these ones as exports, just deprecate them for now. Don't wanna add MUI to Storybook.